spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -7,18 +7,19 @@
 
 RSpec.describe Reek::Configuration::ConfigurationFileFinder do
   describe '.find' do
-    it 'returns the path if it’s set' do
+    it 'returns the file in :path if it’s set' do
```

Comment:
Hm, why the change? `ConfigurationFileFinder#find` doesn’t return a `File`, it returns a `Pathname`.